components/sections/decasteljau/content.zh-CN.md

```diff
@@ -0,0 +1,61 @@
+# de Casteljau 算法
+
+要计算贝塞尔曲线，我们必须遍历 `t` 的所有值，从 `0` 到 `1`，计算权重函数，得到 `x/y` 的值。当曲线很复杂时，计算量变得非常巨大。利用 `de Casteljau` 算法（一种几何画法），你可以轻易地用笔和尺画出曲线。
+
+我们用以下算法来替代微积分算法：
+
+- 把 `t` 看做一个比例，`t=0` 代表线段的 `0%`，`t=1` 代表线段的 `100%`。
+
+- 画出所有点的连线，对 `n` 阶曲线来说可以画出 `n` 条线。
+
+- 在每条线上一定比例长度处做一个记号。比如 `t` 是 `0.2`，就在开始处 `20%` 的地方做个记号。
+
+- 把相邻的记号连线，得到 `n-1` 条线。
+
+- 在这些新得到的线上同样用 `t` 为比例标记。
+
+- 把相邻标记连成线，得到 `n-2` 根线。
+
+- 标记，连线，这样重复下去。
+
+- 直到剩下一根线。这根线上 `t` 的位置就是曲线 `t` 的位置。
+
+<div className="howtocode">
+
+### 如何实现 `de Casteljau` 算法
```

Comment:
If there is no additional markup in the title, then there should be no additional markup in this heading either.